readme.md

```diff
@@ -79,13 +74,22 @@ The streets are imported afterwards from the OSM pbf file with [osm2mimir](https
 
 Finally the POIs are extracted from the PostgreSQL database (thus we have a unified POI handling) using [Fafnir](https://github.com/QwantResearch/fafnir).
 
-### POI API data
+#### Places API data
 
 Idunn does not need its own data import process, but the API depends on:
 
 * the data of the [geocoder](#geocoder_data),
 * Wikipedia data, fetched either on the Wikipedia APIs or in a custom Elasticsearch database.
 
+##### OSM updates <a name="osm_updates"></a>
+
+The world keeps moving and OpenStreetMap data is getting better and better eaeverych day, so we need to update the data in all our components on a regular basis.
```

Comment:
each or every :stuck_out_tongue_winking_eye:

Reply:
each and every ? :roll_eyes: 
wishing for the week end !